src/providers/completer/environment.ts

```diff
@@ -3,9 +3,9 @@ import * as fs from 'fs'
 import {latexParser} from 'latex-utensils'
 
 import type {Extension} from '../../main'
-import type {IProvider, ILwCompletionItem} from './interface'
+import type {IProvider} from './interface'
 import {resolveCmdEnvFile} from './commandlib/commandfinder'
-import {cmdHasOptionalArgs, CmdSignature, getCmdSignature} from './command'
+import {Suggestion} from './command'
```

Comment:
We had better write a comment on why we can use the `Suggention` class for `environment` while it is defined in `./command` . I am not sure where we should write it. It might not be appropriate at this line.

Reply:
This is because every environment can also be inserted using `\envname`. All these commands are computed in the `Environment` class and then called from `Command`

https://github.com/James-Yu/LaTeX-Workshop/blob/db9044cc915d57f5e9013bf5f03cc910c88ae693/src/providers/completer/command.ts#L188

Reply:
> every environments can also be inserted using \envname

I see. We had better write the explanation as a comment somewhere on `environment.ts`.